src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlElement.cs

```diff
@@ -66,7 +66,7 @@ public void Write(StringBuilder strBuilder, DocPosition docPos, AncestralNamespa
             {
                 anc.GetNamespacesToRender(this, attrListToRender, nsListToRender, nsLocallyDeclared);
 
-                strBuilder.Append("<" + Name);
+                strBuilder.Append('<').Append(Name);
```

Comment:
Just to confirm - this won't bug out if _Name_ suddenly becomes null, right?

Reply:
> this won't bug out if Name suddenly becomes null, right?

Correct.
https://github.com/dotnet/runtime/blob/2d6b77fa4eb4811d33f23d0183a12cf2b5d4ab6e/src/libraries/System.Private.CoreLib/src/System/Text/StringBuilder.cs#L807